MapboxNavigation/RouteVoiceController.swift

```diff
@@ -28,10 +28,15 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
      */
     public var volume: Float {
         get {
+            guard let audioPlayer = audioPlayer else {
+                return 1
+            }
             return audioPlayer.volume
         }
         set {
-            audioPlayer.volume = newValue
+            if let audioPlayer = audioPlayer {
+                audioPlayer.volume = newValue
+            }
```

Comment:
I'm making some changes to the volume settings in #186 
I think we should set the volume in UserDefault.
AVAudioPlayer should be optional so when we set the settings:
```swift
settings.volume = newValue
audioPlayer?.volume = newValue
```
 and when AVAudioPlayer is initialized, we grab the volume from the settings.